scripts/docker-compose/docker-compose.yaml

```diff
@@ -280,9 +280,8 @@ services:
       - caddy_config:/config
     networks:
       - opereplay-net
-    environment:
-      - ACME_AGREE=true # Agree to Let's Encrypt Subscriber Agreement
-      - CADDY_DOMAIN=or-foss.rjsh.me
+    env_file:
+      - caddy.env
```

Comment:
Nice catch with the domain name, that was a templating mistake.

We can't have `caddy.env` because when people do `docker-compose up/down` without sourcing common.env, and will happen most because of human errors, we'll have to have
```
environment:
  - ACME_AGREE=true
  - CADDY_DOMAIN=${COMMON_DOMAIN_NAME}
```

When the initial installation command, runs all these env variables will get overriden with the sourced values from common.env and hence, `docker-compose up/down` won't be an issue.